### PR TITLE
Fixes #9890: Remove relay specific conf from rudder apache config files

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9890

### CODE REVIEW

**Comment by amousset** [comment]
Should be merged with https://github.com/Normation/rudder-packages/pull/1166

**Comment by amousset** [comment]
Commit modified

File: rudder-web/src/main/resources/rudder-apache-webapp-ssl.conf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <IfVersion < 2.4>
+    Order deny,allow
+    Allow From all
+  </IfVersion>
+  <IfVersion >= 2.4>
+    Require all granted
+  </IfVersion>
+</Proxy>
+
+<FilesMatch "\.(cgi|shtml|phtml|php)$">
```
Comment (peckpeck) [inline rudder-web/src/main/resources/rudder-apache-webapp-ssl.conf]: Move this bloc to relay-common

File: rudder-web/src/main/resources/rudder-apache-webapp-ssl.conf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  SSLOptions +StdEnvVars
+</FilesMatch>
+<Directory /usr/lib/cgi-bin>
+  SSLOptions +StdEnvVars
+</Directory>
+
+# MSIE 7 and newer should be able to use keepalive
```
Comment (peckpeck) [inline rudder-web/src/main/resources/rudder-apache-webapp-ssl.conf]: move this bloc to webapp-common

File: rudder-web/src/main/resources/rudder-apache-webapp-common.conf [hunk 3]
```diff
@@ -4,8 +4,6 @@
   Header add X-Chrome-Exponential-Throttling "disable"
 </LocationMatch>
 
-DocumentRoot /var/www
-
 # Prevent the apache logs from beeing cluttered by 404 errors
 # due to a missing robots.txt file.
 Alias /robots.txt /opt/rudder/share/load-page/robots.txt
```
Comment (peckpeck) [inline rudder-web/src/main/resources/rudder-apache-webapp-common.conf]: ErrorDocument shoud work properly in relay and we should avoid sending it if it is called in a relay api

**Pull request merged**